zio-http/src/main/scala/zio/http/Handler.scala

```diff
@@ -852,6 +873,14 @@ object Handler {
   def ok: Handler[Any, Nothing, Any, Response] =
     status(Status.Ok)
 
+  def param[A]: ParamExtractorBuilder[A] =
+    new ParamExtractorBuilder[A](())
+
+  class ParamExtractorBuilder[A](val unit: Unit) extends AnyVal {
```

Comment:
Small file organization request :) 
Let's have all these helper classes either inline or in the companion object but not mixed

Reply:
Fixed